simplified-grid/simplified-grid.css

```diff
@@ -0,0 +1,47 @@
+@import url(/style.css);
+
+.grid-container {
+  border: 1px solid var(--dark-black);
+  width: 60%;
+  height: auto;
+  margin: 2rem 0;
+}
+.grid-item {
+  padding: 10px;
```

Comment:
use rem instead of px if not necessary

Reply:
Its a small value, I am assuming it to not to change in responsiveness too. Thats why used px.

Reply:
oh cool then